Lib/test/test_plistlib.py

```diff
@@ -503,6 +503,26 @@ def test_xml_encodings(self):
                 pl2 = plistlib.loads(data)
                 self.assertEqual(dict(pl), dict(pl2))
 
+    def test_dump_invalid_format(self):
+        with self.assertRaises(ValueError):
+            plistlib.dumps({}, fmt="blah")
+
+    def test_load_invalid_file(self):
+        with self.assertRaises(plistlib.InvalidFileException):
+            plistlib.loads(b"these are not plist file contents")
+
+    def test_modified_uid_negative(self):
+        neg_uid = UID(1)
+        neg_uid.data = -1  # dodge the negative check in the constructor
+        with self.assertRaises(ValueError):
+            plistlib.dumps(neg_uid, fmt=plistlib.FMT_BINARY)
+
+    def test_modified_uid_huge(self):
+        neg_uid = UID(1)
+        neg_uid.data = 2 ** 64  # dodge the size check in the constructor
```

Comment:
```suggestion
        huge_uid.data = 2 ** 64  # dodge the size check in the constructor
```

Also, the docs say:

> `data` must be in the range *0 <= data <= 2\*\*64*.

I'm not a `plistlib` expert, but this means that `2 ** 64` should be a valid value. Should the docs be corrected? This can be a separate issue, if so.